keyboards/massdrop/alt/rules.mk

```diff
@@ -31,6 +31,7 @@ HD44780_ENABLE = no         # Enable support for HD44780 based LCDs (+400)
 VIRTSER_ENABLE = no         # USB Serial Driver
 RAW_ENABLE = no             # Raw device
 AUTO_SHIFT_ENABLE = no      # Auto Shift
+TAP_DANCE_ENABLE = yes
```

Comment:
```suggestion
```

Reply:
Committing all this was a mistake, I'll fix it tomorrow.